content/en/tracing/trace_collection/custom_instrumentation/android/otel.md

```diff
@@ -0,0 +1,514 @@
+Trace.enable(traceConfig);
+```
+{{% /tab %}}
+{{< /tabs >}}    
+
+4. Datadog tracer implements the [Open Telemetry standard][18]. Create the `OtelTracerProvider` and register the `OpenTelemetrySdk` in `GlobalOpenTelemetry` as a singleton instance to be used in your application. We recommend for best practices to do this in your `onCreate()` method.
+
+{{< tabs >}}
+{{% tab "Kotlin" %}}
+```kotlin
+GlobalOpenTelemetry.set(object : OpenTelemetry {
+    private val tracerProvider = OtelTracerProvider.Builder()
+        .setService([BuildConfig.APPLICATION_ID])
+        .build()
+
+    override fun getTracerProvider(): TracerProvider {
+        return tracerProvider
+    }
+
+    override fun getPropagators(): ContextPropagators {
+        return ContextPropagators.noop()
+    }
+})
+// and later on if you want to access the tracer provider
+val tracerProvider = GlobalOpenTelemetry.get().getTracer(instrumentationName = "<instrumentation_name>")
+```
+{{% /tab %}}
+{{% tab "Java" %}}
+```java
+```
+{{% /tab %}}
+{{% tab "Java" %}}
+```java
+
+{{% /tab %}} 
+{{% tab "Java" %}}
+```java
+GlobalOpenTelemetry.set(new OpenTelemetry() {
+    private final TracerProvider tracerProvider = new OtelTracerProvider.Builder()
+            .setService(BuildConfig.APPLICATION_ID)
+            .build();
+
+    @Override
+    public TracerProvider getTracerProvider() {
+        return tracerProvider;
+    }
+
+    @Override
+    public ContextPropagators getPropagators() {
+        return ContextPropagators.noop();
+    }
+};
+// and later on if you want to access the tracer provider
+final TracerProvider tracerProvider = GlobalOpenTelemetry.get().getTracer("<instrumentation_name>");       
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+**Note**: If you decide to set the `GlobalOpenTelemetry.set` API you need to make sure that this is only called one single time in your process otherwise OpenTelemetry will throw an exception. In case you want to avoid that you can just create a `TracerProvider` and use it as a singleton in your project.
+
+**Note**: The `setService` method is used to set the service name for the tracer provider. The service name is used to identify the application in the Datadog UI. You can either use the `GlobalOpenTelemetry` to hold a single instance of the `TracerProvider` create your own instance and use it in your application code as needed.
+
+5. Instrument your code with the OpenTelemetry API:
+
+{{< tabs >}}
+{{% tab "Kotlin" %}}
+```kotlin
+val span = tracer.spanBuilder(spanName = "<span_name>").startSpan()
+// do something you want to measure ...
+// ... then, when the operation is finished:
+span.end()
+```
+{{% /tab %}}
+{{% tab "Java" %}}
+```java
+final Span span = tracer.spanBuilder("<span_name>").startSpan();
+// do something you want to measure ...
+// ... then, when the operation is finished:
+span.end();
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+6. (Optional) Set child-parent relationship between your spans:
```

Comment:
After the setup instructions, I think we should break these steps into sections instead like we recently did here: https://docs.datadoghq.com/tracing/trace_collection/custom_instrumentation/java/otel/.

Reply:
I just followed the `iOS` docs that were just merged. I think we should stay consistent in that case:
https://docs.datadoghq.com/tracing/trace_collection/custom_instrumentation/ios/otel/.